src/components/navbar.js

```diff
@@ -1,93 +1,96 @@
-    <Container className="navbar">
-        <Dropdown item text="ABOUT">
-          <Dropdown.Menu>
-            <Dropdown.Item as={Link} to="/about" text="WHAT IS ACSU?" />
-            <Dropdown.Item>OFFICERS</Dropdown.Item>
-          </Dropdown.Menu>
-        </Dropdown>
-
-        <Dropdown item text="EVENTS">
-          <Dropdown.Menu>
-            <Dropdown.Item>CALENDAR</Dropdown.Item>
-            <Dropdown.Item>G-BODY</Dropdown.Item>
-          </Dropdown.Menu>
-        </Dropdown>
-
-        <Dropdown item text="RESOURCES">
-          <Dropdown.Menu>
-            <Dropdown.Item>ACSU README</Dropdown.Item>
-            <Dropdown.Item>RESUME BOOK</Dropdown.Item>
-            <Dropdown.Item>CS WIKI</Dropdown.Item>
-          </Dropdown.Menu>
-        </Dropdown>
-
-        <Menu.Item as="a"
-          name="Sponsor"
-        >
-          SPONSOR
-          </Menu.Item>
-
-        <Menu.Item as="a"
-          name="Hacsu"
-        >
-          HACSU
-        </Menu.Item>
+class Navbar extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            navbarVisible: true
+        }
+    }
 
-        <Menu.Item as="a"
-          name="Hacsu"
-        >
-          JOIN
-        </Menu.Item>
-
-        <Menu.Item as="a" header>
-        <Image
-          className="social-media-icon"
-          src="https://acsu.cornell.edu/img/icons/facebook-icon2.png"
-        />
-        </Menu.Item>
-
-        <Menu.Item as="a" header>
-        <Image
-          className="social-media-icon"
-          src="https://acsu.cornell.edu/img/icons/insta-icon.png"
-        />
-        </Menu.Item>
+    toggleNavbar = () => {
+        this.setState(prevState => ({ navbarVisible: !prevState.navbarVisible }))
+    }
+    
+    resize = () => {
+        let shouldShowNavbar = window.innerWidth > 768
+        if (shouldShowNavbar !== this.state.navbarVisible) {
+            this.setState({ navbarVisible: shouldShowNavbar })
+        }
+    }
 
-      </Container> 
-      </Menu>
-      </div>
-)
+    componentDidMount() {
+        window.addEventListener("resize", this.resize);
+        this.resize();
+    }
+    
+    componentWillUnmount() {
+        window.removeEventListener("resize", this.resize)
+    }
 
-Navbar.propTypes = {
-  siteTitle: PropTypes.string,
-}
+    render() {
+        const { activePage } = this.props;
 
-Navbar.defaultProps = {
-  siteTitle: ``,
+        return (
+            <nav className="navbar">
+                <Icon id="mobile-hamburger" name={ this.state.navbarVisible ? "times" : "bars" } onClick={this.toggleNavbar} />
+                <Link className="logo" to="/">
+                    <Image className="logo-image" src="images/logo_ACSU.png" />
+                    <span>ACSU</span>
+                </Link>
+                <Menu className={ this.state.navbarVisible ? "navbar-links visible" : "navbar-links hidden" } borderless stackable widths={5}>
+                    <Menu.Item 
```

Comment:
i think a potentially better way would be to take advantage of JSX and write a `map` function and factor all the stuff within `<Menu.Item>` out into an array or something. Like `const pages = ['team', 'events', ...]` and then do `pages.map(page => <Menu.Item>...`

Reply:
will fix